content/engineering-practices-fishbowl-dev-cop.md

```diff
@@ -0,0 +1,42 @@
+Title: Fishbowl Dev CoP
+Date: 2023-12-14
+Category: Posts 
+Tags: engineering, technical-excellence, dev-cop, fishbowl
```

Comment:
@MerlinDarkstar dev-cop and fishbowl are not valid tags. Will replace them with event when publishing.